scripts/release-train-issue-template.md

```diff
@@ -43,14 +43,14 @@ Wind down PR queue. There has to be enough time after the last (non-trivial) PR
 - [ ] Make sure there are no stray staging repos on sonatype
 - [ ] Wait until [master build finished](https://travis-ci.org/akka/akka-http/builds/) after merging the release notes
 - [ ] Create a [new release](https://github.com/akka/akka-http/releases/new) with the next tag version (e.g. `v13.3.7`), title and release description linking to announcement, release notes and milestone.
-- [ ] Check that the Travis CI release build has executed successfully
+- [ ] Check that the Travis CI [release build](https://travis-ci.org/akka/akka-http/branches) has executed successfully
 - [ ] Go to https://bintray.com/akka/maven/akka-http and select the just released version
-- [ ] Go to the Maven Central tab and sync with Sonatype
+- [ ] Go to the Maven Central tab and sync with Sonatype. You may need to log in and switch to the 'Old Look' for this.
```

Comment:
Interesting, we should probably investigate to switch to direct sonatype upload using bundles at some point.

Reply:
yeah perhaps - though probably bintray holds on to our snapshots longer? but it would be nice to shave some more steps out of the release process ;)